hotline/images/migrations/0002_auto_20150731_1546.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
Did you modify the image app at all? If not, don't create migrations for them. If you just run makemigrations without arguments, it will eagerly create migrations in all apps, even when they aren't really needed.